_posts/2024-03-01-pyos-newsletter-march-2024.md

```diff
@@ -0,0 +1,56 @@
+
+
+## :purple_heart: Hello, sunPy
+
+[sunPy](https://sunpy.org/), a Python package that provides fundamental tools for accessing, loading, and interacting with solar physics data in Python, has successfully gone through the pyOpenSci open peer review process, which is the first step in our budding partnership with the heliophysics community!
+
+You can read more about sunPy and pyOpenSci in [this lovely blog post](https://sunpy.org/posts/2024/2024-01-24-pyopensci) from Nabil Friej.
+
+## :wave: Say hi to sciform
+
+The [sciform package](https://sciform.readthedocs.io/en/stable/), which is used to convert Python numbers into strings according to a variety of user-selected scientific formatting options, was accepted into the pyOpenSci ecosystem! sciform, like all of the pyOpenSci packages, completed the [pyOpenSci open peer review process](https://www.pyopensci.org/about-peer-review/index.html). And the community support around getting the sciform package review-ready was amazing to see!
+
+Have a package you’ve developed, but aren’t sure it’s in scope for pyOpenSci? Complete a [pre-submission inquiry](https://github.com/pyOpenSci/software-submission/issues/new?assignees=&labels=presubmission&projects=&template=presubmission-inquiry.md&title=) and one of our editors will be in touch!
+
+## :airplane: We're going to [PyCon US](https://us.pycon.org/2024/)
+Will we see you there? Our Executive Director and Founder, [Leah Wasser](https://github.com/lwasser), had her talk about Python packaging for scientists accepted for this May's conference. We've seen a sneak peek of her talk, and guarantee that you won't want to miss it! Be sure to [register today](https://us.pycon.org/2024/accounts/login/?next=/2024/registration/register)!
+
+## :tada: Congratulations, Leah!
+
+Leah Wasser also received a [Better Scientific Software (BSSw) fellowship](https://bssw.io/pages/bssw-fellowship-program) to create lessons around collaborative open science using GitHub. The courses are currently in development, and we'll be sure to let you know once they're ready for registration!
+
+[BSSw](https://bssw.io/) provides a central hub for the community to address pressing challenges in software productivity, quality, and sustainability.
+
+## :books: New lessons are LIVE!
+pyOpenSci released new and updated lessons in our [Python Packaging Guide](https://www.pyopensci.org/python-package-guide/index.html), designed to help scientists both make their Python code more shareable, as well as how to create a Python package. Be sure to check them out and let us know what you think!
```

Comment:
can we link to the tutorials page here instead so people are directed to the tutorials https://www.pyopensci.org/python-package-guide/tutorials/intro.html

Reply:
fixed